src/data/content/quality.ts

```diff
@@ -77,8 +77,90 @@ const techContent = [
+        data: [
+          {
+            name: "Selenium Webdriver",
+            description: `Selenium is an open-source testing framework. It comes with built-in libraries for web and mobile app testing. Axe core can be integrated with selenium test written with Java, Python, and Ruby. Example selenium implementation can be found here:<br><br><a href="https://github.com/dequelabs/axe-core-maven-html/tree/develop">https://github.com/dequelabs/axe-core-maven-html/tree/develop/</a>`,
+          },
+          {
+            name: "Jasmine",
+            description: `Jasmine is an open-source unit testing framework for JavaScript. Axe can be integrated with Jasmin unit test framework. The example can be found here:<br><br><a href="https://github.com/dequelabs/axe-core/tree/develop/doc/examples/jasmine">https://github.com/dequelabs/axe-core/tree/develop/doc/examples/jasmine</a>`,
+          },
+          {
+            name: "Mocha",
+            description: `Mocha is a JavaScript unit test framework for Node.js programs, featuring browser support, asynchronous testing, test coverage reports, and use of any assertion library. Axe can be integrated with the Mocha unit testing framework. The example on how to integrate with mocha can be found here:<br><br><a href="https://github.com/dequelabs/axe-core/tree/develop/doc/examples/mocha">https://github.com/dequelabs/axe-core/tree/develop/doc/examples/mocha</a>`,
+          },
+          {
+            name: "Cucumber Ruby",
+            description: `Automated accessibility testing using ruby can be integrated with axe. More details can be found here in the ruby example: <br><br><a href="https://github.com/dequelabs/axe-core-gems">https://github.com/dequelabs/axe-core-gems</a>`,
+          },
+          {
+            name: "QUnit Testing Framework",
+            description: `This example demonstrates how to use axe with the QUnit unit testing framework<br><br><a href="https://github.com/dequelabs/axe-core/tree/develop/doc/examples/qunit">https://github.com/dequelabs/axe-core/tree/develop/doc/examples/qunit </a>`,
+          },
+        ],
+      },
+      {
+        name: "Use Cases",
+        intro: "",
+        data: [
+          {
+            name: "Quality Engineering Framework Tech Test",
+            description: `An example of axe core library integration with Java Gradle Selenium Project which demonstrates the ability to use automation test for accessibility testing of online booking website can be found in repo <a href="https://github.com/camg-capco/tech-test">https://github.com/camg-capco/tech-test</a> Please request access if you want to see the detail as it not yet made public. `,
```

Comment:
@okyd-capco we can move this to the View Projects section. For that the GitHub repo has to be made public and tagged with the technology - Axe.